internal/controller/kyma_controller/kyma_test.go

```diff
@@ -79,12 +90,23 @@ var _ = Describe("Kyma enable one Module", Ordered, func() {
 
 	It("should result in Kyma becoming Ready", func() {
 		By("checking the state to be Processing")
-		Eventually(GetKymaState, Timeout, Interval).
-			WithArguments(kyma.GetName()).
-			Should(Equal(string(v1beta2.StateProcessing)))
+		Eventually(IsKymaInState, Timeout, Interval).
+			WithContext(ctx).
+			WithArguments(kyma.GetName(), kyma.GetNamespace(), controlPlaneClient, v1beta2.StateProcessing).
+			Should(Succeed())
 
 		By("having created new conditions in its status")
-		Eventually(GetKymaConditions, Timeout, Interval).WithArguments(kyma.GetName()).ShouldNot(BeEmpty())
+		Eventually(func() error {
+			createdKyma, err := GetKyma(ctx, controlPlaneClient, kyma.GetName(), kyma.GetNamespace())
```

Comment:
I think this code is less readable than the original one, which was a one-liner. Generally I think we should keep the test code as short as possible and possibly reflecting the "given/when/then" BDD style. IMO this code, although simple, is long enough to deserve it's own function.

Reply:
the problem of [original one](https://github.com/kyma-project/lifecycle-manager/pull/956/files#diff-886762db4cf11d4f54f47a446185f94f18c2777d10def986b78b5509c0707982L101)is it expected return empty condition when get kyma get error, which is not solid enough, but I agree it deserve a function.